discordBot.js

```diff
@@ -43,6 +43,45 @@ class DiscordBot {
   stop() {
     return this.client.destroy();
   }
+
+  // send files to discord channel given the file path and channel
+  sendFile(filePath, channelId, callback){
+    this.client.on('ready', () => {
+      var attachment = new MessageAttachment(filePath);
+      return callback(this.client.channels.cache.get(channelId).send(attachment));
+    })    
+  }
+
+  //sends a link to a google form to the specifies channel
+  sendGoogleForm(googleFormUrl, channelId, callback){
+    this.client.on('ready', () => {
+      return callback(this.client.channels.cache.get(channelId).send(googleFormUrl));
```

Comment:
This looks a bit wired.
1. the `sendGoogleForm(...)` function doesn't return any object
2. we have on('ready') callback which is fired once `discordBot` is login
3. in the `ready` callback we call another callback function with a promise parameter which is called when the discord bot is login.
It's just not very javascript-like style.
I would just go ahead
```
return this.client.channels.cache.get(channelId).send(googleFormUrl);
```

Reply:
returning that leads to undefined

Reply:
fixed it so using then and callback returns the promise after it has been resolved. still so you can call it discordBot.sendfile(filepath, channelid, function(res) => {console.log(res))